app/const.js

```diff
@@ -1,62 +1,62 @@
 export const SESSIONSCHEMA = {
-    "type": "jsonschema",
```

Comment:
Very small, but I left this as `"`s so that it was still valid JSON for copy pasting purposes. :shrug: